test/e2e/diagram.navigation.fit.test.ts

```diff
@@ -28,36 +28,36 @@ import { AvailableTestPages, PageTester } from '@test/shared/visu/bpmn-page-util
 
 class FitImageSnapshotConfigurator extends ImageSnapshotConfigurator {
-  override getConfig(param: {
+  override getConfig(parameter: {
     fileName: string;
     buildCustomDiffDir: (config: MatchImageSnapshotOptions, fitType: FitType, margin?: number) => string;
     fitType: FitType;
     margin?: number;
   }): MatchImageSnapshotOptions {
-    const config = super.getConfig(param);
-    config.customSnapshotsDir = FitImageSnapshotConfigurator.buildSnapshotFitDir(config.customSnapshotsDir, param.fitType, true, param.margin ? param.margin : 0);
-    config.customDiffDir = param.buildCustomDiffDir(config, param.fitType, param.margin);
+    const config = super.getConfig(parameter);
+    config.customSnapshotsDir = FitImageSnapshotConfigurator.buildSnapshotFitDirectory(config.customSnapshotsDir, parameter.fitType, true, parameter.margin ? parameter.margin : 0);
+    config.customDiffDir = parameter.buildCustomDiffDir(config, parameter.fitType, parameter.margin);
     return config;
   }
 
-  private static buildSnapshotFitDir(parentDir: string, fitType: FitType, withMargin = false, margin?: number): string {
-    const typeDir = join(parentDir, `type-${fitType}`);
+  private static buildSnapshotFitDirectory(parentDirectory: string, fitType: FitType, withMargin = false, margin?: number): string {
+    const typeDirectory = join(parentDirectory, `type-${fitType}`);
 
     if (!withMargin) {
-      return typeDir;
+      return typeDirectory;
     }
-    return join(typeDir, `margin-${margin == null || margin < 0 ? 0 : margin}`);
+    return join(typeDirectory, `margin-${margin == null || margin < 0 ? 0 : margin}`);
   }
 
-  static buildOnLoadDiffDir(config: MatchImageSnapshotOptions, fitType: FitType, withMargin = false, margin?: number): string {
-    const onLoadDir = join(config.customDiffDir, 'on-load');
-    return FitImageSnapshotConfigurator.buildSnapshotFitDir(onLoadDir, fitType, withMargin, margin);
+  static buildOnLoadDiffDirectory(config: MatchImageSnapshotOptions, fitType: FitType, withMargin = false, margin?: number): string {
+    const onLoadDirectory = join(config.customDiffDir, 'on-load');
+    return FitImageSnapshotConfigurator.buildSnapshotFitDirectory(onLoadDirectory, fitType, withMargin, margin);
   }
 
-  static buildAfterLoadDiffDir(config: MatchImageSnapshotOptions, afterLoadFitType: FitType, onLoadFitType: FitType): string {
-    const afterLoadDir = join(config.customDiffDir, 'after-load');
-    const snapshotFitTypeDir = FitImageSnapshotConfigurator.buildSnapshotFitDir(afterLoadDir, afterLoadFitType);
-    return join(snapshotFitTypeDir, `on-load_type-${onLoadFitType}`);
+  static buildAfterLoadDiffDirectory(config: MatchImageSnapshotOptions, afterLoadFitType: FitType, onLoadFitType: FitType): string {
+    const afterLoadDirectory = join(config.customDiffDir, 'after-load');
+    const snapshotFitTypeDirectory = FitImageSnapshotConfigurator.buildSnapshotFitDirectory(afterLoadDirectory, afterLoadFitType);
```

Comment:
**suggestion:** rename into `snapshotFitDirectory` for consistency with the called method and for clarity